.github/workflows/test.yml

```diff
@@ -102,3 +102,35 @@ jobs:
           # Use `haswell` instead of `native` due to some GitHub Actions
           # runners not supporting some `avx512` instructions.
           RUSTFLAGS: "-C target-cpu=haswell"
+
+  coverage:
+    name: Coverage
+    runs-on: ubuntu-latest
+    # Don't collect coverage if tests failed
+    needs: test
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Set excluded examples
+        id: excluded-examples
+        run: echo "::set-output name=out::$(grep -h '^name' examples/**/Cargo.toml | cut -d'"' -f2 | xargs -I '{}' printf '--exclude {} ')"
+
+      - name: Collect coverage
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          args: '--ignore-tests --workspace --all-features ${{ steps.excluded-examples.outputs.out }}'
+        env:
+          # Use `haswell` instead of `native` due to some GitHub Actions
+          # runners not supporting some `avx512` instructions.
+          RUSTFLAGS: "-C target-cpu=haswell"
```

Comment:
nearly every rust invocation needs this, cleaner to set it globally

Reply:
i'm not aware of a way to do this